linear_search/linear_search.cpp

```diff
@@ -0,0 +1,41 @@
+
+#include <iostream>
+using namespace std;
+
+// Function to perform linear search
+// Parameters:
+//   - long long arr[]: an array to search in
+//   - long long n: size of the array
+//   - long long x: key to search for
+// Returns:
+//   - long long: index of the key if found, -1 otherwise

```

Comment:
**Format - 1**
```
// Function to perform linear search
// Parameters:
//   - long long arr[]: an array to search in
//   - long long n: size of the array
//   - long long x: key to search for
// Returns:
//   - long long: index of the key if found, -1 otherwise
```
**Format - 2**
```
/*
 Function to perform linear search
 Parameters:
   - long long arr[]: an array to search in
   - long long n: size of the array
   - long long x: key to search for
 Returns:
  - long long: index of the key if found, -1 otherwise
*/
```